src/lit-element.ts

```diff
@@ -104,35 +107,29 @@ export class LitElement extends UpdatingElement {
     }
     // Take care not to call `this.getStyles()` multiple times since this
     // generates new CSSResults each time.
-    // TODO(sorvell): Since we do not cache CSSResults by input, any
```

Comment:
I think this comment needs to stay. It's referring to there not being a cache from TemplateStringsArray to CSSResult, so we end up creating a new CSSResult and therefore new CSSStyleSheet each time we evaluate the same `css` tag. This can happen when people use `static get styles()` rather than `static styles =`, and the accessor is called multiple times.